felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -823,6 +877,25 @@ func (m *bpfEndpointManager) updateWEPsInDataplane() {
 	})
 }
 
+func (m *bpfEndpointManager) wepIfaceReady(name string, ready bool) {
+	iface := m.nameToIface[name]
+	if iface.info.ifindex == 0 {
+		return
+	}
+
+	flgReady := ifstate.FlgReady
+	if !ready {
+		flgReady = 0
+	}
+
+	m.ifStateMap.SetDesired(
+		ifstate.NewKey(uint32(iface.info.ifindex)).AsBytes(),
```

Comment:
Please can you add a utility method to actually do the map write.  Best not to have two places that build up the keys/flags in case we add a new flag later.